docs/walkthroughs/provision-keycloak-apb.adoc

```diff
@@ -1,22 +1,22 @@
 [[provision-keycloak-service]]
-= Provision KeyCloak Service
+= Provision http://www.keycloak.org/documentation.html[KeyCloak] Service
 
 == Provision
 Choose the type of service you would like to provision:
 
 * Keycloak service
-* Linked Keycloak service
+* Shared Keycloak service
 
 Go to the OpenShift web console. Open the *Mobile* Tab > *Services* from the service catalog and select the Keycloak service.
 This should open the Keycloak service provision wizard. Press the *Next* button and provide the configuration for the KeyCloak service
 
 === KeyCloak service
-This service type can be shared between different Keycloak services.
+This service type can be shared between shared between different projects
 
-*The configuration would ask for the following parameters:*
+*The configuration will ask for the following parameters:*
 
 * *Add to Project(required)*: Create a new project or point to an existing project
-* *Keycloak admin username(required)*: The username of the admin user for the Keycloak web console
+* *Keycloak admin username(required)*: This user should be able to administer the realm or create new realms
```

Comment:
would it be possible to add a space before each opening bracket on all list points, just to make it look consistent?